example/lib/main.dart

```diff
@@ -101,11 +118,45 @@ class _MyAppState extends State<MyApp> {
     }
   }
 
+  Future <Uint8List> makeBuffer(String path) async
+  {
+
+    try
+    {
+      File file = File(path);
+      file.openRead();
+      var contents = await file.readAsBytes ();
+      print ('The file is ${contents.length} bytes long.');
+      return contents;
+    } catch (e)
+    {
+      print(e);
+      return null;
+    }
+  }
+
   void startPlayer() async{
     try {
-      String path = await flutterSound.startPlayer(this._path);
-      await flutterSound.setVolume(1.0);
-      print('startPlayer: $path');
+      String path = null;
+      if (_playFromBuffer == 0) // Do we want to play from buffer or from file ?
```

Comment:
The indentation can go upward as described in [effective dart](https://dart.dev/guides/language/effective-dart/design#do-annotate-when-dart-infers-the-wrong-type).

Below is common format.
![image](https://user-images.githubusercontent.com/27461460/70851814-88cd8f00-1edd-11ea-948c-2c521ce7eaa4.png)